src/test/LocationList.test.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { LocationList } from "@components";
+
+describe("<LocationList />", () => {
+  let Location1 = [
```

Comment:
const instead of let, and please make the mock data more realistic, eg proper lat/long values